ENGAGEHF/VitalsManager/VitalsUnit.swift

```diff
@@ -0,0 +1,37 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import HealthKit
+
+
+enum VitalsUnit: CustomStringConvertible {
```

Comment:
Could we potentially replace this with HKUnit instead? or provide a computed property that returns the HKUnit rather than doing string conversions?

Reply:
I think the issue is that HKUnit doesn't do `BPM` and elements that we would need for the visualizations.